infra/ansible/playbooks/telemetry.yaml

```diff
@@ -0,0 +1,183 @@
+- name: Run Setup playbook
+  ansible.builtin.import_playbook: setup.yaml
+  vars:
+    host: telemetry
+
+- name: Run Caddy playbook
+  ansible.builtin.import_playbook: caddy.yaml
+  vars:
+    host: telemetry
+    ini_file: ini/config-telemetry.ini
+
+- name: Run Prometheus playbook
+  ansible.builtin.import_playbook: prometheus.yaml
+  vars:
+    host: telemetry
+    prometheus_version: "2.53.2"
+    ini_file: ini/config-telemetry.ini
+
+- name: Run Alert Manager playbook
+  ansible.builtin.import_playbook: alert_manager.yaml
+  vars:
+    host: telemetry
+    alert_manager_version: 0.27.0
+    ini_file: ini/config-telemetry.ini
+
+- name: Run Grafana playbook
+  ansible.builtin.import_playbook: grafana.yaml
+  vars:
+    host: telemetry
+    ini_file: ini/config-telemetry.ini
+
+- name: Run OpenTelemetry playbook
+  ansible.builtin.import_playbook: open_telemetry.yaml
+  vars:
+    host: telemetry
+    open_telemetry_version: "0.113.0"
+
+- name: Run Cassandra playbook
+  ansible.builtin.import_playbook: cassandra.yaml
+  vars:
+    host: telemetry
+    ini_file: ini/config-telemetry.ini
+    cassandra_version: 5.0.2
+
+- name: Run Jaeger playbook
+  ansible.builtin.import_playbook: jaeger.yaml
+  vars:
+    host: telemetry
+    ini_file: ini/config-telemetry.ini
+    jaeger_version: 1.63.0
+
+- name: Run Postgres 16 playbook
+  ansible.builtin.import_playbook: postgres.yaml
+  vars:
+    host: telemetry
+    ini_file: ini/config-telemetry.ini
+
+- name: Run Elixir playbook
+  ansible.builtin.import_playbook: elixir.yaml
+  vars:
+    host: telemetry
+
+- name: Run Go playbook
+  ansible.builtin.import_playbook: go.yaml
+  vars:
+    host: telemetry
+
+- name: Setup Telemetry
+  hosts: telemetry
+
+  tasks:
+  - name: Make sure /etc/default/tailscaled exists
+    become: true
+    file:
+      path: /etc/default/tailscaled
+      state: touch
+    vars:
+      ansible_ssh_user: "{{ admin_user }}"
+
+  - name: Add permit to tailscale for caddy
+    become: true
+    lineinfile:
+      path: /etc/default/tailscaled
+      line: TS_PERMIT_CERT_UID=caddy
+      state: present
+    vars:
+      ansible_ssh_user: "{{ admin_user }}"
+    notify:
+        - Restart caddy
+        - Restart tailscale
+      
+  - name: Clone Aligned repository
+    ansible.builtin.git:
+      repo: https://github.com/yetanotherco/aligned_layer.git
+      dest: /home/{{ ansible_user }}/repos/telemetry/aligned_layer
+      #version: v0.12.0
+      version: staging
```

Comment:
Temporary fix to allow compilation, since `v.012.0` has a missing comma inside a JSON.

Should be pinned to a future release.